app/views/shared/_get_ready.html.erb

```diff
@@ -1,43 +1,16 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
-</head>
-
-<body>
-  <hr>
-  <div class="container-fluid">
-    <h1 class="text-center"> Get Ready for the Day </h1>
-    <h3 class="text-center"> Know Your Tools</h3>
-    <br>
-    <div class="container marketing">
-      <div class="row">
-        <div class="col-md-4">
-          <img class="img-circle center" src="../../assets/images/slack_logo.jpg" alt="slack logo" width="140" height="140">
-          <h2 class="text-center"> Slack <h2>
-            <p class ="lead">
-            "Slack brings all your communication together in one place. It's real-time messaging, archiving and search for modern teams."
-          </p>
-
-        </div>
-        <div class="col-md-4">
-          <img class="img-circle center" src="../../assets/images/chrome_logo.jpg" alt="chrome logo" width="140" height="140">
-          <h2 class="text-center"> Chrome <h2>
-            <p class ="lead">
-            "Google Chrome browser is an open source program for accessing the World Wide Web and running Web-based applications."
-          </p>
-
-        </div>
-        <div class="col-md-4">
-          <img class="img-circle center" src="../../assets/images/atom_logo.jpg" alt="atom logo" width="140" height="140">
-          <h2 class="text-center"> Atom <h2>
-            <p class ="lead">
-            "Atom is a free and open-source text and source code editor support for plug-ins written in Node.js, developed by GitHub."
-          </p>
-
-        </div>
-      </div>
-    </div>
+<section class="row text-center">
+  <h2>Get Ready for the Day</h2>
+  <p class="lead">Know Your Tools</p>
+  <div class="col-md-4">
+    <img src="" height="140px" width="140px" alt="Small picture" class="img-circle" />
```

Comment:
dont use inline `height` nor `width` attributes. slap an extra `class` on it, and style that with CSS